webtool/frontend/src/components/cost-section.tsx

```diff
@@ -0,0 +1,80 @@
+import {FunctionComponent} from "react"
+import {DataList, Heading} from "@radix-ui/themes"
+import { PiMoneyLight } from "react-icons/pi"
+
+export const CostDisplay: FunctionComponent<{ artifact: any }> = ({artifact}) => {
+    return (
+        <div>
+            <CostHeading />
+            <DataList.Root>
+                <DataList.Item>
+                    <DataList.Label minWidth="88px">Kosten per kWh</DataList.Label>
+                    <DataList.Value>{artifact.costsPer_kWh}</DataList.Value>
+                </DataList.Item>
+                <DataList.Item>
+                    <DataList.Label minWidth="88px">Aanschaf [€]</DataList.Label>
+                    <DataList.Value>{artifact.buy_ct}</DataList.Value>
+                </DataList.Item>
+                <DataList.Item>
+                    <DataList.Label minWidth="88px">Rente</DataList.Label>
+                    <DataList.Value>{artifact.income_r * 100} %</DataList.Value>
+                </DataList.Item>
+                <DataList.Item>
+                    <DataList.Label minWidth="88px">Afschrijvingsperiode [jaar]</DataList.Label>
+                    <DataList.Value>{artifact.writingPeriod_y}</DataList.Value>
+                </DataList.Item>
+                <DataList.Item>
+                    <DataList.Label minWidth="88px">Onderhoudskosten [€/jaar]</DataList.Label>
+                    <DataList.Value>{artifact.additionalCosts_cty}</DataList.Value>
+                </DataList.Item>
+            </DataList.Root>
+        </div>
+    )
+}
+
+const CostHeading = () => (
+    <Heading as="h3" style={{paddingBottom: ".5rem"}}>
+        <PiMoneyLight />
+        &nbsp;
+        Kosten
+    </Heading>
+);
+
+export const CostSection: FunctionComponent<{onCostChange: (key: any, value: any) => void}> = ({onCostChange}) => {
+    return (
+        <div>
+            <CostHeading/>
+            <div>
+                <label htmlFor="costsPer_kWh">Kosten per kWh</label>
```

Comment:
Please add to the label or to right of the input that it is in euros so people don't enter cents.

Reply:
I've done it before. is this what is expected?
<img width="588" alt="Screenshot 2024-09-26 at 2 49 41 PM" src="https://github.com/user-attachments/assets/201cd03c-f534-4ecc-8c16-c335596b7008">

Reply:
:+1: